README.md

```diff
@@ -63,6 +63,8 @@ Major features include:
   for stateful models
 - Provides [Backend API](https://github.com/triton-inference-server/backend) that
   allows adding custom backends and pre/post processing operations
+- Supports model architecture agnostic python scripts as custom
+  python based backends.
```

Comment:
Can we expand on it? Link to more detailed docs in python backend? Maybe we can also point to vLLM backend as an example somewhere.

Reply:
More docs in backend repo are comming. I am fixing pre-commit hooks:  https://github.com/triton-inference-server/backend/pull/88